pkg/applyconfiguration/testdata/cronjob/api/v1/groupversion_info.go

```diff
@@ -27,9 +27,10 @@ import (
 )
 
 var (
-	GroupName     = "testdata.kubebuilder.io"
-	GroupVersion  = schema.GroupVersion{Group: GroupName, Version: "v1"}
-	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	GroupName          = "testdata.kubebuilder.io"
+	GroupVersion       = schema.GroupVersion{Group: GroupName, Version: "v1"}
+	SchemeGroupVersion = GroupVersion
+	SchemeBuilder      = runtime.NewSchemeBuilder(addKnownTypes)
```

Comment:
This fixes the issue, but we don't have unit test coverage that this actually compiles, right?

Reply:
Perfect